server/commands.go

```diff
@@ -82,3 +85,39 @@ func ResetAll() *cobra.Command {
 
 	return resetAll
 }
+
+// ResetState removes all the block state stored in dymint
+func ResetState() *cobra.Command {
+	return &cobra.Command{
+		Use:     "reset-state",
+		Aliases: []string{"reset_state"},
+		Short:   "Remove all the data and WAL",
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+
+			config := server.GetServerContextFromCmd(cmd).Config
+			var paths []string
+			appdb := filepath.Join(config.DBDir(), "application.db")
+			dymintdb := filepath.Join(config.DBDir(), "dymint")
+			settlementdb := filepath.Join(config.DBDir(), "settlement")
+			paths = append(paths, appdb, dymintdb, settlementdb)
+			for _, path := range paths {
+				err := removePath(path)
+				if err != nil {
+					fmt.Printf("error removing %s with error %s\n", path, err)
+				}
+			}
+			return nil
+		},
+	}
+}
+
+func removePath(path string) error {
+	if cmtos.FileExists(path) {
+		if err := os.RemoveAll(path); err == nil {
+			fmt.Println("Removed application.db", "file", path)
```

Comment:
`application.db` logging is probably wrong. `path` should be enough

Reply:
fixed